test/test-pages/tumblr/expected-metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Minecraft 1.8 - The Bountiful Update - Minecraft 1.8 - The Bountiful Update",
```

Comment:
```
does tumblr always repeat the title?
```
In general, no. But that is from the original page. I used the page because the blog is one of the top tumblr blogs.